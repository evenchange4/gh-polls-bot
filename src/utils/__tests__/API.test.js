// @flow
const API = require('../API');

jest.mock('request-promise', () => () => ({ id: 'MOCK_ID' }));

it('should resolve id with API response', async () => {
  expect(await API.addPoll(['1', '2'])).toMatchSnapshot();
});
