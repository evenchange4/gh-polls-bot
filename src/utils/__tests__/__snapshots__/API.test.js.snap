// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should resolve id with API response 1`] = `"MOCK_ID"`;
